shared/actions/settings.tsx

```diff
@@ -379,13 +379,67 @@ const deleteAccountForever = (state, action: SettingsGen.DeleteAccountForeverPay
 }
 
 const loadSettings = () =>
-  RPCTypes.userLoadMySettingsRpcPromise().then(
-    settings =>
-      settings.emails &&
-      SettingsGen.createLoadedSettings({
-        emails: I.List(settings.emails.map(row => Constants.makeEmailRow(row))),
-      })
-  )
+  RPCTypes.userLoadMySettingsRpcPromise().then(settings => {
```

Comment:
is there a ticket to add `settings.phones`?

Reply:
As in, to the service side of this RPC? I'm not sure. I don't know of one.

Reply:
Can you make one if it doesn't exist?

Reply:
it now exists and is done, thanks to michal